packages/auth/plugin/src/ios/urlTypes.ts

```diff
@@ -84,8 +91,10 @@ export function setUrlTypesForCaptcha({
     );
   }
 
-  const reversedClientId = getReversedClientId(googleServiceFilePath);
-  addUriScheme(config, reversedClientId);
+  if(reversedClientIDExists(googleServiceFilePath)) {
+    const reversedClientId = getReversedClientId(googleServiceFilePath);
+    addUriScheme(config, reversedClientId);
+  }
```

Comment:
may be worth putting an else branch in here with a console.warn saying something like `reversed client id field not found in GoogleServices-Info.plist. Google Sign-In requires this is - if you need Google Sign-In, enable it and re-download your plist file`

Or something like that - it is kind of wordy but I'd rather have an extra console.log during build then a bunch of subtle issues in the github issue tracker later from Expo people that are lost, and we've forgotten why it was even needed ourselves...